pentaho-karaf-features/src/main/resources/pentaho-karaf-features-server.xml

```diff
@@ -83,7 +83,7 @@
         <bundle start-level="20">mvn:pentaho/pentaho-ss2-proxies/${project.version}</bundle>
         <bundle start-level="50">mvn:pentaho/pentaho-proxy-spring4/${project.version}</bundle>
 
-        <bundle>mvn:pentaho/pentaho-requirejs-osgi-manager/${project.version}</bundle>
+        <feature version="${project.version}">pentaho-requirejs-osgi-manager</feature>
```

Comment:
We generally place <feature> dependencies at the top of a feature definition. No worries